scripts/normalize_text.py

```
@@ -0,0 +1,97 @@
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+import re
+import sys
+from pathlib import Path
+
+FIND_URL_PATTERN = re.compile(r'https?://[^\s<>"]+')
+REPLACEMENTS = {
+    # Space (U+0020)
+    " ": " ",  # No-Break Space (U+00A0)
```

Ruff (0.15.10)

[warning] String contains ambiguous ` ` (NO-BREAK SPACE). Did you mean ` ` (SPACE)?

(RUF001)